src/SDK/Client/Platform/methods/names/register.ts

```diff
@@ -92,11 +91,12 @@ export async function register(this: Platform,
     );
 
```

Comment:
Let's add a wait 500 here to make sure that all nodes processed the previous block?